### DISCUSSION TITLE
Alpaca模型在线对战gradio代码

### DISCUSSION THREAD

**Discussion by DarrenIm** [ymcui/Chinese-LLaMA-Alpaca#776] [General]
hi，很棒的工作！
http://chinese-alpaca-arena.ymcui.com/ 模型在线对战设计的很有意思！请问相关gradio代码可以开源吗？有个人项目做类似的事情，想借鉴一下，十分感激！

**Comment by ymcui** [comment]
你好，近期在做LLaMA-2相关工作，暂时没有时间整理代码，后续可能会考虑开放。
我们的demo是参考fastchat设计的，你可以参考：
- https://chat.lmsys.org/?arena 以及 https://colab.research.google.com/drive/1RAWb22-PFNI-X1gPVzc927SGUdfr6nsR?usp=sharing
你也可以参考这个：
- https://huggingface.co/spaces/openaccess-ai-collective/rlhf-arena
